### DISCUSSION TITLE
Problem with the dataset slice conversion format

### DISCUSSION THREAD

**Discussion by diejump** [obss/sahi#1033] [Q&A] [answered]
I use the VisdDrone2019 dataset and slice with the "slice _ coco" function and I change the parameter "out_ext" to ".jpg ", I found that the final sliced image would automatically disappear , but use ". png " works fine, I checked the code and found when " out_ext "set as". jpg ", the " image_pil.save (slice_file_path, quality= "keep") int " _ export _ single _ slice_image " function run error, and did not save the picture correctly. How should this change?

**Comment by fcakyon** [comment, marked as answer]
Thanks for bringing this up @diejump ! Its fixed with: https://github.com/obss/sahi/pull/1037
